src/exm-types.h

```diff
@@ -17,6 +17,8 @@ typedef enum
     EXM_EXTENSION_STATE_OUT_OF_DATE = 4,
     EXM_EXTENSION_STATE_DOWNLOADING = 5,
     EXM_EXTENSION_STATE_INITIALIZED = 6,
+    EXM_EXTENSION_STATE_DISABLING = 7,
+    EXM_EXTENSION_STATE_ENABLING = 8,
```

Comment:
Not sure if this is the proper way to fix them.

Reply:
I think this is fine - I've seen this be done in other GLib projects so happy with it